external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -40,13 +40,13 @@ object SchemaConverters {
   case class SchemaType(dataType: DataType, nullable: Boolean)
```

Comment:
if you target to expose and document `SchemaConverters`, it should probably be marked as `@DeveloperAPI` with a proper version in the doc.

Reply:
Got it. The code is updated.